composer.json

```diff
@@ -50,6 +50,7 @@
         "liip/imagine-bundle": "1.3.0",
         "imagine/imagine": "0.6.2",
         "monolog/monolog": "1.22.0",
+        "ocramius/proxy-manager": "2.0.4",
```

Comment:
@damien-carcel on our branch we don't care about the CI with PHP5.6 ;)
but yes, why not remove it :)